clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -332,6 +332,429 @@ TEST_F(FormatTestComments, UnderstandsSingleLineComments) {
   verifyNoCrash(StringRef("/*\\\0\n/", 6));
 }
 
+TEST_F(FormatTestComments, InsertsSpaceAfterOpeningBlockComment) {
+  FormatStyle Style = getLLVMStyle();
+  Style.SpaceInComments.AfterOpeningComment =
+      FormatStyle::CommentSpaceMode::Always;
+
+  verifyFormat("foo(/* comment */);", "foo(/*comment */);", Style);
+  verifyFormat("/* comment */", "/*comment */", Style);
+  verifyFormat("/* comment before code */\n"
+               "int x;",
+               "/*comment before code */\n"
+               "int x;",
+               Style);
+  verifyFormat("/* \n"
```

Comment:
Do we really want that? Many editors remove trailing whitespace on save, which would happen here. And even though the option says to add a space, I don't think we should add a space before a line break. We never do that, the line break is in this regard also a space.

Similar for the closing part and also for parameter comments.